test/unit/cli-domain-handle-dependencies-get-compiler.js

```diff
@@ -33,7 +32,7 @@ describe('cli : domain : handle-dependencies : get-compiler', () => {
       }
     ).default;
 
-    getCompiler(_.cloneDeep(options))
+    getCompiler(options)
```

Comment:
why are we not cloning anymore?

Reply:
because options are created per test on this same function and only used to being passed to this function, so it was absolutely useless cloning

Reply:
makes sense